pkg/controllers/disruption/consolidation.go

```diff
@@ -219,8 +226,14 @@ func (c *consolidation) computeSpotToSpotConsolidation(ctx context.Context, cand
 	instanceTypeOptionsWithSpotOfferings :=
 		results.NewNodeClaims[0].NodeClaimTemplate.InstanceTypeOptions.Compatible(results.NewNodeClaims[0].Requirements)
 
-	// Possible replacements that are lower priced than the current candidate
-	results.NewNodeClaims[0].NodeClaimTemplate.InstanceTypeOptions = filterByPrice(instanceTypeOptionsWithSpotOfferings, results.NewNodeClaims[0].Requirements, candidatePrice)
+	var incompatibleRequirementKey string
+	// Possible replacements that are lower priced than the current candidate and the requirement that is not compatible with minValues
+	results.NewNodeClaims[0].NodeClaimTemplate.InstanceTypeOptions, incompatibleRequirementKey =
+		filterByPriceWithMinValues(instanceTypeOptionsWithSpotOfferings, results.NewNodeClaims[0].Requirements, candidatePrice)
+
+	if len(incompatibleRequirementKey) > 0 {
+		return Command{}, pscheduling.Results{}, fmt.Errorf("minValues requirement is not met for %s", incompatibleRequirementKey)
```

Comment:
Is this an error or should this be handled in the same way that we handle other times that we don't have instance type options (see L238)

Reply:
We can merge these 2. I was thinking if this would bring more clarity since `minValues` is something that is controlled by the customer and would inform them to relax the condition ?

For the sake of decreasing the noise in the logs, we can skip this altogether and just rely on L238 to bring a closure and probably publish an event specifically for this ?

Reply:
Merged this and created a separate event for not meeting the minValues requirement.